oauth2-server/src/main/java/com/clouway/oauth2/idtoken/IdToken.java

```diff
@@ -0,0 +1,123 @@
+package com.clouway.oauth2.idtoken;
+
+import com.clouway.oauth2.jwt.Jwt;
+
+/**
+ * @author Vasil Mitov <v.mitov.clouway@gmail.com>
+ */
+public final class IdToken {
```

Comment:
It's corresponding Claims has different fields than the JWT. Was wondering if I can just change the JWT ClaimSet fields and add the ones I need for the Token to have the needed fields.